### DISCUSSION TITLE
Se llena el disco

### DISCUSSION THREAD

**Discussion by Pollo0531YT** [bucanero/pkgi-ps3#11] [Q&A] [answered]
Hola bucanero
Muchas gracias por compartir esta genial "app" 

Tengo un problema, al crear el espacio ficticio del pkg, se va rebajando el espacio en el disco duro, pero cuando cancelo el espacio sigue reducido 
Voy a la carpeta TMP/pkgi y no hay nada para borrar y recuperar el espacio 

Ayúdame, como recupero el espacio, no sé qué borrar

**Comment by yungripo** [comment]
Prueba a entrar en dev_hdd0/packages, ahi te saldran carpetas con muchos 0000000, dentro de una de ellas estará el archivo con nombre NPEB, NPUB etc... que te hayas empezado a descargar

**Comment by bucanero** [comment, marked as answer]
en la carpeta `tmp/pkgi` solo se guardan temporalmente las descargas directas

para las descargas background, los archivos temporales se generan en `vsh/task/10000xxx`. Los puedes borrar de alli, ~~o simplemente reiniciar el sistema, y el propio PS3 eliminará todas las tareas "incompletas" que encuentre.~~

**Comment by Pollo0531YT** [comment]
Gracias amigo 
Lo reinicie hasta más de 10 veces y no borra nada "incompleto" como dices ahí
La solución fue  borrar las carpetas con los 000xxx como dices del juego que estaba descargando y solucionado 
Volvió todos mis gb Jaja 

Muchas gracias
